queries.sql

```diff
@@ -43,3 +43,17 @@ SELECT neutered, COUNT(*) AS escape_count  FROM animals WHERE escape_attempts >
  SELECT species , MIN(weight_kg) AS min_weight , MAX(weight_kg) AS  max_weight FROM animals GROUP BY species;
  SELECT species , MIN(weight_kg) AS min_weight , MAX(weight_kg) AS  max_weight FROM animals GROUP BY species;
  SELECT species, AVG(escape_attempts) AS avg_escape_attempts FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
+
+
```

Comment:
- [ ] Great work but it would be nice if you submit your pull request on the right link on your dashboard 👍